### ISSUE TITLE
ValueError: Need to enable queue to use generators. SD-CN error on Paperspace

### ISSUE THREAD

**Issue by CoqueTornado** [TheLastBen/fast-stable-diffusion#2298]
On paperspace there is an error called "ValueError: Need to enable queue to use generators."

This appears when I try the SD-CN video animator.

Somewhere said to make a git pull or an git remote set-url origin https://github.com/AUTOMATIC1111/stable-diffusion-webui

even more something like this:

//git reset -- hard
//git checkout modules/script_callbacks.py
//git reset HEAD modules/script_callbacks.py
//git checkout  modules/ui.py
//git reset HEAD modules/script_callbacks.py

Tried also to force the reinstall the dependences to true. Nothing worked.
Should I start with an empty SD folder renaming the Sd to SD-old ? is that the only solution?

Thank you for your support

**Comment by TheLastBen** [comment]
I tried to fix that extension without success, try contacting the extension dev

**Comment by CoqueTornado** [comment]
oh, maybe it is related with something about the gpu and the new torch version like happens with the control net 1.1? 

thank you for all your support and work

**Comment by TheLastBen** [comment]
Yes, I remember now, one of the dependencies required by the extension isn't compatible with torch 2
